src/main/java/org/launchcode/workforce/controller/TimeStampController.java

```diff
@@ -0,0 +1,48 @@
+package org.launchcode.workforce.controller;
+
+import org.launchcode.workforce.model.TimeStamp;
+import org.launchcode.workforce.service.TimeStampService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping("/timeClock")
+public class TimeStampController {
+
+    private TimeStampService timeStampService;
+
+    public TimeStampController(TimeStampService timeStampService) {
+        this.timeStampService = timeStampService;
+    }
+
+    //how will you label between clock in and clock out
+    @PostMapping("/clock")
+    @ResponseStatus(HttpStatus.OK)
+    TimeStamp add(@RequestBody TimeStamp timeStamp) {
```

Comment:
consider simpler args coming from the frontend to limit writing javascript (since javascripts is harder).